### ISSUE TITLE
v2.1.0.beta.5

### ISSUE THREAD

**Issue by StephenMcAvoy** [neverhood311/Stop-motion-OBJ#85]
It works well. The sequences can also be used in Arrays
I see that the imported mesh sequence can no longer be split into objects. Any particular reason?

**Comment by neverhood311** [comment]
Yeah, I tried enabling the Split by Group option for an OBJ sequence with multiple groups and it only keeps the first object as part of the sequence. The remainder of the objects are loaded but not handled by the addon. Right now this is by design. Stop Motion OBJ is not set up to handle multi-object OBJs. Maybe in the the future but that's not something I can tackle right now.

**Comment by StephenMcAvoy** [comment]
I thought it might be something like that. It won’t be much of a problem to me as I won’t want to edit the geometry only the materials. I think that the Wavefront (.obj) importer needs some more work since the 2.8 version broke a few things. None of the alpha blending is being activated and there is a uv mapping issue I can’t track down.
I was wrong about using an array with STOP Motion. Modifiers cannot be applied to multi-user data.
